### PR TITLE
Add tagsClass options on node and global

### PR DESCRIPTION
**Pull request by Zehir**
This allow replacing class of tags on node. Per tag and per node or globally. The default value is unchanged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -679,7 +689,7 @@ Triggers `nodeExpanded` event; pass silent to suppress events.
 Returns an array of matching node objects.
 
 ```javascript
-$('#tree').treeview('findNode', ['Parent', 'text']);
+$('#tree').treeview('findNodes', ['Parent', 'text']);
```
Comment (skateman) [inline README.md]: Is this relevant to the PR?

Comment (Zehir) [inline README.md]: Not really but the correct method is “findNodes” and not “findNode”.
If I try this example I get: “No such method : findNode”

Comment (skateman) [inline README.md]: OK, but next time put it into a separate PR :wink:

Comment (Zehir) [inline README.md]: Ok NP

**Comment by skateman** [comment]
:+1:

**Comment by Zehir** [comment]
Do this merge update the minified version?

**Comment by Zehir** [comment]
@skateman Do this merge update the dist/minified version, bower version and npm version?

**Comment by skateman** [comment]
@Zehir nope, I'll push it into a release soon.

**Comment by skateman** [comment]
@Zehir here you go: https://github.com/patternfly/patternfly-bootstrap-treeview/releases/tag/v2.1.3
